src/opm/parser/eclipse/EclipseState/Schedule/WellConnections.cpp

```diff
@@ -116,14 +116,19 @@ namespace {
 } // anonymous namespace
 
     WellConnections::WellConnections(int headIArg, int headJArg) :
+        m_connections(),
```

Comment:
No harm - and I am happy to merge in the current form, but initializing `m_connections()` is not necessary?

Reply:
It's not clear what value to set for `m_connections`.  This is the moral equivalent of the default constructor for `WellConnections` and the contents of `m_connections` will be generated by `addConnection()` calls&mdash;be it directly or through `loadCOMPDAT()`.  In my opinion it makes sense to just default initialise `m_connections` here.  I guess we could just not mention the data member here (as before), but it is declared ahead of `headI` and `headJ` so some compilers might choose to warn about initialisation out of order.

Reply:
OK - but I though that a `std::vector<T>` was well defined default constructed? If the argument is mainly: 

> it is declared ahead of headI and headJ so some compilers might choose to warn about initialisation out of order.

I would actually say that changing the declaration order is a better solution?

Reply:
> I though that a `std::vector<T>` was well defined default constructed?

It is.  I'm just calling the default constructor
```C++
vector<T>::vector()
```
here.

> If the argument is mainly:
> 
> > it is declared ahead of `headI` and `headJ` so some compilers might choose to warn about initialisation out of order.
> 
> I would actually say that changing the declaration order is a better solution?

Agreed.  I'll update the PR accordingly.